apps/privacy-panel/locales/privacypanel.en-US.properties

```diff
@@ -88,24 +94,26 @@ passphrase-invalid = You cannot use special characters!
 passphrase-different = Confirmation must match passphrase!
 pin-empty = Passcode lock is empty!
 pin-invalid = Wrong Passcode lock!
+passcode-long = Passcode should be 4 digits!
 pin-different = Wrong Passcode lock!
-sim-invalid = Wrong SIM PIN!
+sim-invalid = Wrong SIM PIN! {{ value }} tries left!
```

Comment:
I didn't see this in my previous reviews. Was it there from the beginning?

Not only it needs a new string ID, it also needs proper plural form support.